apps/src/sites/studio/pages/congrats/index.js

```diff
@@ -47,21 +44,21 @@ $(document).ready(function () {
       isHocTutorial,
       isPlCourse,
       isK5PlCourse,
+      courseNames: [courseName],
     });
 
   ReactDOM.render(
     <Provider store={store}>
       <Congrats
         certificateId={certificateId}
-        tutorial={tutorial}
+        tutorial={courseName}
```

Comment:
If it's possible that we pass down multiple course names, should this be `{[courseName]}` as well?

Reply:
yes, sort of! I'm actually changing it to be a list of objects (I need the course name and the course path), but I'll do that in a future PR (you can see the in progress work [here](https://github.com/code-dot-org/code-dot-org/pull/56791/files), but that PR is still in draft). But, I didn't want to add an Amplitude attribute then change it two days later.